components/service-operator/apis/database/v1beta1/postgres_types.go

```diff
@@ -181,19 +185,24 @@ func (p *Postgres) GetStackTemplate() (*cloudformation.Template, error) {
 			DBClusterIdentifier:     cloudformation.Ref(PostgresResourceCluster),
 			DBInstanceClass:         coalesce(p.Spec.AWS.InstanceType, DefaultClass),
 			Engine:                  Engine,
-			EngineVersion:           EngineVersion,
 			PubliclyAccessible:      false,
 			DBParameterGroupName:    cloudformation.Ref(PostgresResourceParameterGroup),
 			Tags:                    tags,
 			DBSubnetGroupName:       cloudformation.Ref(DBSubnetGroupNameParameterName),
 			DeleteAutomatedBackups:  false,
 			CACertificateIdentifier: "rds-ca-2019",
+			EngineVersion:           p.Spec.AWS.EngineVersion,
```

Comment:
EngineVersion could be nil here, but previously it was never `nil`.  Is this an issue? Is this deliberate?  (Is this even the whole point of the PR? I don't know 🤷 )

Reply:
Yes it's deliberate ... _originally_ it was nil (defaults to "latest version") ... setting it to `nil` for existing resources that do not specify it is required for backwards compatibility.